_topic_map.yml

```diff
@@ -39,12 +39,13 @@ Topics:
 ---
 Name: Release notes
 Dir: release_notes
-Distros: openshift-enterprise
+Distros: openshift-enterprise, openshift-dedicated
```

Comment:
@sheriff-rh The space here might be breaking it.

Reply:
good catch @adellape  ! just updated.